cloudinit/config/cc_write_files_deferred.py

```diff
@@ -2,31 +2,36 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 
-"""Defer writing certain files"""
+"""
+Write Files Deferred
+------------
+**Summary:** Defer writing certain files
 
-from cloudinit import util
-from cloudinit.config.cc_write_files import DEFAULT_DEFER, write_files
+This module is based on `'Write Files' <write-files>`__, and
+will handle all files from the write_files list, that have been
+marked as deferred and thus are not being processed by the
+write-files module.
 
-# meta is not used in this module, but it remains as code documentation
-#
-# id: cc_write_files_deferred'
-# name: 'Write Deferred Files
-# distros: ['all'],
-# frequency: PER_INSTANCE,
-# title:
-#    write certain files, whose creation as been deferred, during
-#    final stage
-# description:
-#    This module is based on `'Write Files' <write-files>`__, and
-#    will handle all files from the write_files list, that have been
-#    marked as deferred and thus are not being processed by the
-#    write-files module.
-#
-#    *Please note that his module is not exposed to the user through
-#    its own dedicated top-level directive.*
+*Please note that his module is not exposed to the user through
+its own dedicated top-level directive.*
+"""
 
-# Not exposed, because related modules should document this behaviour
-__doc__ = None
+from cloudinit import util
+from cloudinit.config.cc_write_files import DEFAULT_DEFER, write_files
+from cloudinit.config.schema import MetaSchema
+from cloudinit.distros import ALL_DISTROS
+from cloudinit.settings import PER_INSTANCE
+
+# This module isn't documented
+meta: MetaSchema = {
+    "id": "cc_write_files_deferred",
+    "name": "Write Files Deferred",
+    "title": "Defer writing certain files",
+    "description": __doc__,
+    "distros": [ALL_DISTROS],
```

Comment:
If you feel excited about cleaning out cruft, we could also drop any  `distros= ["all"]` module attributes use ALL_DISTROS instead
```
$ fgrep -r "distro" cloudinit/config/cc_* | grep '"all"'
cloudinit/config/cc_bootcmd.py:distros = ["all"]
cloudinit/config/cc_chef.py:distros = ["all"]
cloudinit/config/cc_keys_to_console.py:distros = ["all"]
cloudinit/config/cc_locale.py:distros = ["all"]
cloudinit/config/cc_mcollective.py:distros = ["all"]
cloudinit/config/cc_migrator.py:distros = ["all"]
cloudinit/config/cc_mounts.py:distros = ["all"]
cloudinit/config/cc_update_etc_hosts.py:distros = ["all"]
cloudinit/config/cc_update_hostname.py:distros = ["all"]
cloudinit/config/cc_users_groups.py:    "distros": ["all"],
cloudinit/config/cc_write_files.py:    "distros": ["all"],
```